CODEOWNERS

```diff
@@ -45,4 +45,5 @@ components/sup* @christophermaier @reset @fnichol @baumanj @raskchanky
 support @fnichol @baumanj @elliott-davis @raskchanky
 tools @baumanj @chefsalim @christophermaier @eeyun @raskchanky @fnichol
 www @cnunciato @mgamini @raskchanky
```

Comment:
Because of the way CODEOWNERS works, the most specific lines should come first, and the most generic should come last, otherwise @kagarmoe will get pinged for review of a whole bunch of things that she doesn't care about.

@kagarmoe Do you mind moving the line you added _above_ the existing line that reads `www @cnunciato @mgamini @raskchanky`?